src/components/Agentathon/Card/index.tsx

```diff
@@ -0,0 +1,12 @@
+import { Typography } from '@mui/material'
+import css from './style.module.css'
+export default function Card({ title, image }: { title: string; image: { src: string; alt: string } }) {
```

Comment:
Nit: let's add a new line between `import` and `export`.
```suggestion
import css from './style.module.css'

export default function Card({ title, image }: { title: string; image: { src: string; alt: string } }) {
```

Reply:
Updated